db/migrate/20160525143440_thredded_user_topic_follows.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+class ThreddedUserTopicFollows < ActiveRecord::Migration
```

Comment:
As this will be in the v0.5 release, let's move to upgrade_migrations/...v04_to_05